phy/cluster/manual/session.py

```diff
@@ -99,6 +100,10 @@ def on_open():
     # File-related actions
     # -------------------------------------------------------------------------
 
+    @property
+    def has_unsaved_changes(self):
+        return self._is_dirty
```

Comment:
well it would be better to test if the spike_clusters and cluster_groups arrays are the same or not 
planned for the future...